### DISCUSSION TITLE
Level 70 Boss Target Dummy?

### DISCUSSION THREAD

**Discussion by Cooties-cc** [chromiecraft/chromiecraft#4755] [Suggestions]
One thing that's bothered me since we got our first raids, was that no city had a level ?? target dummy that had the same avg values (armor mainly) as bosses in the current raid tier. If I wanted to test out different builds and specs, it was difficult to get an accurate reading, since if you went with the only ?? dummy in the area, which is the wotlk one, it's armor value is around 10k. AQ40 bosses by comparison only averaged around 6k armor. This also meant the level 60 normal dummies weren't very accurate too (armor and calcs are diff than lvl 63 target). 

I don't think this is too much to ask to implement a equivalent TBC boss valued dummy to say perhaps shatt city somewhere so players can test out builds. Even as a custom change, the impact is very low, but has high gains for the community.

**Comment by 55Honey** [comment]
Hi @Cooties-cc ,
Thanks for the suggestion. Alongside the dummies for 19, 29, 39 and 49, i think we can look into one more of them.

**Reply by heyitsbench** [reply]
Could even use sniffs from the Classic PTR as a baseline. 👀
